ui/src/app/utils/regex.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * This method executes a search for a match between a regular expression and a specified string.
+ * @param content string that you want to perform search
+ * @param pattern regex pattern
+ * @param flags regex flag
+ * @returns Boolean -- if there is a match it returns true else false
+ */
+export const contains = (content:string,pattern:string,flags?:string) => {
```

Comment:
contains is very generic name, make the utility name redable and align to the context